tests/test_lab_config.py

```diff
@@ -51,6 +57,34 @@ def test_ilab_config_show_key_general_yaml(cli_runner: CliRunner) -> None:
     assert actual_output == expected_yaml_output
 
 
+def test_ilab_config_show_key_remove_comments(cli_runner: CliRunner) -> None:
+    cfg = configuration.get_default_config()
+    key_config_commented_map = configuration.config_to_commented_map(cfg.general)
+
+    yaml_stream = ruamel.yaml.compat.StringIO()
```

Comment:
Logic could be shared with https://github.com/instructlab/instructlab/pull/2602/files in a common method